mail_parent_recipient/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This modules set the default recipient in an email to the first partner having
+an email set following the parents tree. If no parent with an email is found,
+it reverts to the popup asking to set an email for the current partner.
```

Comment:
"reverts bla bla" I don't see anything like that. If you rely on std machinery then just mention that you rely on it.